lib/api/routes/exercises.rb

```diff
@@ -12,7 +13,7 @@ class Exercises < Core
           result = Homework.new(current_user).all.to_json
           LifecycleEvent.track('fetched', current_user.id)
           result
-        rescue Exception => e
+        rescue StandardError => e
```

Comment:
I explicitly want to handle exception here, not standard error, in order to send it to bugsnag and provide a semi-friendly message. There's some syntax to explicitly allow certain violations, right?